infrastructure/modules/k8s/chaos-mesh/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "chaos-mesh" {
```

Comment:
You are alrady in the module chaos_mesh, so I would just call the output chart_name and inline the value.

Reply:
done